src/components/common/Directors.vue

```diff
@@ -1413,31 +1322,45 @@ export default class Directors extends Mixins(CommonMixin, DateMixin, DirectorMi
 
   /** Emits an event containing this component's paid change state. */
   @Emit('directorsPaidChange')
-  private emitDirectorsChange (val: boolean): void { }
+  private emitDirectorsPaidChange (): boolean {
+    return this.isDirectorsPaidChange()
+  }
 
   /** Emits an event containing this component's free change state. */
   @Emit('directorsFreeChange')
-  private emitDirectorsFreeChange (val: boolean): void { }
+  private emitDirectorsFreeChange (): boolean {
+    return this.isDirectorsFreeChange()
+  }
 
   /** Emits an event containing the director form's validity. */
   @Emit('directorFormValid')
-  private emitDirectorFormValid (val: boolean): void { }
-
-  /** Emits an event containing the complete directors list. */
-  @Emit('allDirectors')
-  private emitAllDirectors (val: DirectorIF[]): void { }
-
-  /** Emits an event containing the active directors list. */
-  @Emit('activeDirectors')
-  private emitActiveDirectors (val: DirectorIF[]): void { }
+  private emitDirectorFormValid (): boolean {
+    return this.directorFormValid
+  }
 
   /** Emits an event that indicates whether a director edit is in progress. */
   @Emit('directorEditAction')
-  private emitDirectorEditInProgress (val: boolean): void { }
+  private emitDirectorEditInProgress (): boolean {
+    return this.directorEditInProgress
+  }
 
   /** Emits an event that indicates a director compliance warning has been triggered. */
   @Emit('complianceDialogMsg')
-  private emitcomplianceDialogMsg (val: AlertMessageIF): void { }
+  private emitcomplianceDialogMsg (): AlertMessageIF {
+    return this.complianceMsg
+  }
+
+  /** Emits original addresses object to the parent page. */
+  @Emit('original')
+  private emitOriginalDirectors (): DirectorIF[] {
```

Comment:
I'd argue THIS component should not be telling the parent about the original directors list, but rather that the parent should be telling this component (if it even needs to know).
But that's for another day.